ipn/local.go

```diff
@@ -184,6 +179,118 @@ func (b *LocalBackend) SetDecompressor(fn func() (controlclient.Decompressor, er
 	b.newDecompressor = fn
 }
 
+func (b *LocalBackend) setClientStatus(newSt controlclient.Status) {
```

Comment:
let's rename `newSt` to just `st` while you're doing this move. (I've been meaning to rename that a few times but never quite got around to it)